challenge.js

```diff
@@ -58,4 +59,6 @@
 
 // ANSWER
 
+https:developer.mozilla.org/en-US/docs/Web/API/Node/appendChild
```

Comment:
Right on. By the way, the URL protocol is followed by `://`, not just the colon.

Since you mentioned `document.createElement` in Q1, here is a snippet showing it used in conjunction with `Node.appendChild` to create a brand-new HTML element and attach it to the DOM at a location of our choice:

```js
var div = document.querySelectorAll("body > div")[0]; // the zeroth element of the set of all divs that are immediate children of the body
var e = document.createElement('textarea'); // this textarea element is not on the DOM yet
div.appendChild(e); // mission accomplished
```

1